src/components/AutoComplete/AutoComplete.test.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import AutoComplete from './AutoComplete';
+
+test('AutoComplete is working !!', () => {
+  const bsSize = 'large';
+  const align = 'justify';
+  const clearButton = true;
+  const multiple = true;
+  const allowNew = true;
+  const options = [
```

Comment:
Can you move this list into the `mock` file?

Reply:
Or maybe even better, use a small list (3 items) here? so the snapshot will also be lean

Reply:
+100 on this.  Using max three items would be better for testing.